src/autopas/containers/verletListsCellBased/verletListsCells/traversals/VLCSlicedTraversal.h

```diff
@@ -51,32 +46,36 @@ class VLCSlicedTraversal
    * @param interactionLength cutoff + skin
    * @param cellLength length of the underlying cells
    * @param pairwiseFunctor The functor that defines the interaction of two particles.
+   * @param dataLayout
+   * @param useNewton3
+   * @param typeOfList indicates the type of neighbor list as an enum value, currently only used for getTraversalType
    */
   explicit VLCSlicedTraversal(const std::array<unsigned long, 3> &dims, PairwiseFunctor *pairwiseFunctor,
-                              double interactionLength, const std::array<double, 3> &cellLength)
-      : SlicedLockBasedTraversal<ParticleCell, PairwiseFunctor, dataLayout, useNewton3, false>(
-            dims, pairwiseFunctor, interactionLength, cellLength),
+                              double interactionLength, const std::array<double, 3> &cellLength,
+                              DataLayoutOption dataLayout, bool useNewton3, ContainerOption::Value typeOfList)
+      : SlicedLockBasedTraversal<ParticleCell, PairwiseFunctor>(dims, pairwiseFunctor, interactionLength, cellLength,
+                                                                dataLayout, useNewton3, false),
+        VLCTraversalInterface<typename ParticleCell::ParticleType, NeighborList>(typeOfList),
         _functor(pairwiseFunctor) {}
 
   void traverseParticlePairs() override;
 
-  [[nodiscard]] DataLayoutOption getDataLayout() const override { return dataLayout; }
-
-  [[nodiscard]] bool getUseNewton3() const override { return useNewton3; }
-
   [[nodiscard]] TraversalOption getTraversalType() const override {
-    switch (typeOfList) {
+    switch (this->_typeOfList) {
       case (ContainerOption::verletListsCells):
         return TraversalOption::vlc_sliced;
       case (ContainerOption::pairwiseVerletLists):
         return TraversalOption::vlp_sliced;
+      default:
+        autopas::utils::ExceptionHandler::exception("Traversal was created with an unsupported neighborlist type: " +
+                                                    ContainerOption::getOptionNames()[this->_typeOfList]);
```

Comment:
```suggestion
        autopas::utils::ExceptionHandler::exception("Traversal was created with an unsupported neighborlist type: {}",
                                                    this->_typeOfList.to_string());
```
The array access would result in a segfault if the value is weird. `to_string` handles this well.

Reply:
done in [7d70a16](https://github.com/AutoPas/AutoPas/pull/814/commits/7d70a16ddb368103159095ff87e2a094e7f40e72)